doc/api/util.md

```diff
@@ -130,6 +128,9 @@ _Note: usage of util.inherits() is discouraged. Please use the ES6 `class` and
 `extends` keywords to get language level inheritance support. Also note that
 the two styles are [semantically incompatible][]._
 
+* `constructor` {Function}
+* `superConstructor` {Function}
+
```

Comment:
Yeah, but I continued reading and you added those for the deprecated APIs as well. It would be more consistent then to have this <- change of mind. Thx.